pyk4a/pyk4a.cpp

```diff
@@ -23,6 +24,8 @@ extern "C" {
     #define MAX_DEVICES 32
     device_container devices[MAX_DEVICES];
 
+    const char* capsule_playback_name = "pyk4a playback handle";
```

Comment:
Is this required? We didn't use for the other capsules. I wonder if it makes a difference.

Reply:
It helps in repr. I.e. if you print PyK4APlayback._handle you will see something like `<capsule object "pyk4a playback handle" at 0x7f61b19c19f0>` instead `<capsule object at 0x7f61b19c19f0>`

It will helps you identify which capsule